tests/testem.js

```diff
@@ -25,9 +25,11 @@ fluid.defaults("fluid.tests.testem", {
         tests:   "%infusion/tests"
     },
     testemOptions: {
-        skip: "PhantomJS,Opera,Safari",
+        skip: "PhantomJS,Opera,Safari,Chrome",
```

Comment:
Is skipping "Chrome" required with the headless browser args specified on line 32 below?

Reply:
I believe so, since Chrome and Headless Chrome are separate launchers - otherwise I think testem will launch both.
The l32 argument forces Firefox to run in headless mode as well, which is otherwise not possible without setting an environment variable. May be temporary, but testing that Firefox tests will pass in headless mode under CI.